src/Raven.Client/Documents/Operations/Indexes/ResetIndexOperation.cs

```diff
@@ -10,35 +10,40 @@ namespace Raven.Client.Documents.Operations.Indexes
     public sealed class ResetIndexOperation : IMaintenanceOperation
     {
         private readonly string _indexName;
+        private readonly bool _isSideBySide;
 
-        public ResetIndexOperation(string indexName)
+        public ResetIndexOperation(string indexName, bool isSideBySide = false)
```

Comment:
I think you need a separate, dedicated constructor with two parameters for backward compatibility. CC @ppekrol

Reply:
`isSideBySide` -> `asSideBySide` maybe? (and they fix that in order places as well)

Reply:
Added separate constructor and renamed parameter